workflow-dev/tpl/generate_params.toml

```diff
@@ -17,37 +17,37 @@
 # - azure: Store persistent data in Azure's object storage
 # - gcs: Store persistent data in Google Cloud Storage
 # - minio: Store persistent data on in-cluster Minio server
-storage = "minio"
+storage = "{{ env "STORAGE_TYPE" | default "minio"}}"
 
 [minio]
 org = "deisci"
 pullPolicy = "Always"
 dockerTag = "canary"
 
 [s3]
-accesskey = "YOUR KEY HERE"
-secretkey = "YOUR SECRET HERE"
+accesskey = "{{ env "AWS_ACCESS_KEY" }}"
+secretkey = "{{ env "AWS_SECRET_KEY" }}"
 # Any S3 region
-region = "us-west-1"
+region = "{{ env "REGION" | default "us-west-1" }}"
 # Your buckets.
-registry_bucket = "your-registry-bucket-name"
-database_bucket = "your-database-bucket-name"
-builder_bucket = "your-builder-bucket-name"
+registry_bucket = "{{ env "REGISTRY_BUCKET" | randAlpha 5 }}"
+database_bucket = "{{ env "DATABASE_BUCKET" | randAlpha 5 }}"
+builder_bucket = "{{ env "BUILDER_BUCKET" | randAlpha 5 }}"
```

Comment:
since you prefixed the `accessKey` and `secretKey` env vars with `AWS_`, can you prefix these also with `AWS_`?